README.md

```diff
@@ -142,7 +142,8 @@ int main(int argc, char**argv) {
    // The call is asynchronously executed on your custom sink.
    const int k10MBInBytes = 10 * 1024 * 1024;
    std::future<void> received = sinkHandle->call(&LogRotate::setMaxLogSize, k10MBInBytes);
-   
+   // call this if you want to rotate log in program manually 
```

Comment:
Nitpicking: maybe consider changing the sentence for a better flow
Possibly something like:
` You can manually trigger a rotate of the log `